src/Turtle/Prelude.hs

```diff
@@ -228,6 +228,29 @@ module Turtle.Prelude (
     , gibibytes
     , tebibytes
 
+    -- * File status
+    , PosixCompat.FileStatus
+    , stat
+    , lstat
+    , PosixCompat.deviceID
+    , PosixCompat.fileID
+    , PosixCompat.fileMode
+    , PosixCompat.linkCount
+    , PosixCompat.fileOwner
+    , PosixCompat.fileGroup
+    , PosixCompat.specialDeviceID
```

Comment:
For all the functions from `specialDeviceID` and up I'd suggest removing them until somebody needs them.  My reasoning is that it's not clear what more opinionated and idiomatic type to wrap them in, and we won't know the answer to that until somebody needs to use them.